### PR TITLE
Fix FPC-only address payload

### PR DESCRIPTION
**Pull request by bryce-mcmath**
### Please check if the PR fulfills these requirements:

- [x] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
Bug fix

### CODE REVIEW

**Review by stephanieharrison-maximus — APPROVED** [review]
Looks good to me

File: jha/src/services/api-service.js [hunk 1]
```diff
@@ -43,6 +44,18 @@ class ApiService {
...
       spouse: null, // Set below.
     };
+    
+    // only mail address is required for FPCare-only submissions
+    if (onlyApplyingForFPCare) {
+      jsonPayload.addressLine1 = formState.mailAddressLine1 || null;
+      jsonPayload.addressLine2 = formState.mailAddressLine2 || null;
+      jsonPayload.addressLine3 = formState.mailAddressLine3 || null;
+      jsonPayload.city = formState.mailCity || null;
+      jsonPayload.postalCode = formState.mailPostalCode || null;
```
Comment (harrymaynard-maximus) [inline jha/src/services/api-service.js:54]: You need to strip the spaces from the postal code (use the `stripSpaces` helper).

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: jha/src/services/api-service.js (hunk 1 above)
Comment (bryce-mcmath) [reply jha/src/services/api-service.js]: Done, rerequested your review

File: jha/src/services/api-service.js [hunk 2]
```diff
@@ -47,13 +47,13 @@ class ApiService {
...
-      jsonPayload.postalCode = formState.mailPostalCode;
-      jsonPayload.provinceOrState = formState.mailProvince;
-      jsonPayload.country = formState.mailCountry;
+      jsonPayload.addressLine1 = formState.mailAddressLine1 || null;
+      jsonPayload.addressLine2 = formState.mailAddressLine2 || null;
+      jsonPayload.addressLine3 = formState.mailAddressLine3 || null;
+      jsonPayload.city = formState.mailCity || null;
+      jsonPayload.postalCode = formState.mailPostalCode || null;
+      jsonPayload.provinceOrState = formState.mailProvince || null;
+      jsonPayload.country = formState.mailCountry || null;
```
Comment (harrymaynard-maximus) [inline jha/src/services/api-service.js:56]: I think the country might also require the use of a substring to truncate the value to 25 chars.

**Review by jlangy — APPROVED** [review]

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good!

**Pull request merged**